### ISSUE TITLE
Moodle course summary problem

### ISSUE THREAD

**Issue by germanvaleroelizondo** [EduardoKrausME/moodle-mod_certificatebeautiful#23]
Hi,
 I have created a few test certificates. They are beautiful indeed, but I can not see the page 2 with the Moodle course summary (the Moodle course does have a nice summary).

Is there a limit in the characters or formatting for the Moodle course summary to be shown?

**Comment by germanvaleroelizondo** [comment]
![imagen](https://github.com/user-attachments/assets/a32fb618-10e1-4ba7-a35c-cff97cc0703c)

**Comment by germanvaleroelizondo** [comment]
¿No debería ser 'summary' con dos emes?

**Comment by eduardokraus** [comment]
Sí, esta página debería mostrar el resumen, pero solo está mostrando el número cero.

**Comment by germanvaleroelizondo** [comment]
Gracias por las muy rápidas y eficaces mejoras.

**Comment by germanvaleroelizondo** [comment]
The new plugin version now includes exactly the Moodle course summary.
Thanks a lot.

**Comment by eduardokraus** [comment]
Showwww

**Comment by germanvaleroelizondo** [comment]
![imagen](https://github.com/user-attachments/assets/da4548ce-1647-4792-ade4-be1024671cc6)
